src/jumpstart/jumpstartLinkHandler.test.ts

```diff
@@ -42,26 +45,41 @@ describe('jumpstartLinkHandler', () => {
     const result = await jumpstartLinkHandler(
       networkConfig.defaultNetworkId,
       contractAddress,
-      privateKey,
-      mockAccount2
+      mockPrivateKey,
+      mockUserAddress
     )
 
     expect(result).toEqual(['0xHASH'])
     expect(fetchWithTimeout).toHaveBeenCalledTimes(1)
     expect(fetchWithTimeout).toHaveBeenCalledWith(
-      `${networkConfig.walletJumpstartUrl}?index=1&beneficiary=${mockAccount}&signature=0xweb3-signature&sendTo=${mockAccount2}&assetType=erc20&networkId=celo-alfajores`,
+      `${networkConfig.walletJumpstartUrl}?index=1&beneficiary=${mockBeneficiary}&signature=${mockSignature}&sendTo=${mockUserAddress}&assetType=erc20&networkId=celo-alfajores`,
       expect.any(Object),
       expect.any(Number)
     )
   })
 
   it('throws an error if all funds were already claimed', async () => {
-    mockErc20ClaimsCall.mockResolvedValueOnce({ claimed: true })
-    mockErc20ClaimsCall.mockResolvedValueOnce({ claimed: true })
-    mockErc20ClaimsCall.mockRejectedValue(new Error('execution reverted'))
+    jest
```

Comment:
nit: it looks like this mock implementation is identical to the previous one? could this be shared?

Reply:
ah, it's the same except for one line! the first mock returns a true then false for the claims, and this one is false false 🙈